sphinx/source/conf.py

```diff
@@ -78,12 +78,15 @@
 from glob import glob
 import os
 import jpyutil
+import deephaven_internal
```

Comment:
@jmao-denver We should use DHaaL to simplify sphinx config.  I created a ticket.  https://github.com/deephaven/deephaven-core/issues/2654